README.md

```diff
@@ -1,22 +1,45 @@
-# Dash0 Kubernetes Operator
+# Dash0 Operator
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/dash0-operator)](https://artifacthub.io/packages/search?repo=dash0-operator)
 
-The Dash0 Kubernetes Operator makes observability for Kubernetes _easy_.
+The Dash0 Operator makes observability for Kubernetes _easy_.
 Install the operator into your cluster and create a Dash0 monitoring resource to get OpenTelemetry data flowing from your applications and
 infrastructure to Dash0.
 
+The Dash0 operator is currently available as a technical preview.
+
 ## Description
 
-The Dash0 Kubernetes operator enables gathering OpenTelemetry data from your workloads for a selection of supported
+The Dash0 operator enables gathering OpenTelemetry data from your workloads for a selection of supported
 runtimes, automatic log collection and metrics.
 
-The Dash0 Kubernetes operator is currently available as a technical preview.
+### Distributed tracing
+
+Auto-instrumentation if supported for the following runtimes:
+
+* Node.js 18 and beyond, using [Dash0 Node.js OpenTelemetry Distribution](https://github.com/dash0hq/opentelemetry-js-distribution)
+
+For more information on how the Dash0 operator automatically traces your applications, see the [Automatic Workload Instrumentation](https://artifacthub.io/packages/helm/dash0-operator/dash0-operator#automatic-workload-instrumentation) section of the Dash0 operator Helm chart documentation.
+
+### Metrics
```

Comment:
I think we also should add a section for log collection here.

Reply:
Done in d398150